addons/point_of_sale/models/product.py

```diff
@@ -33,3 +33,21 @@ class ProductUom(models.Model):
     _inherit = 'product.uom'
 
     is_pos_groupable = fields.Boolean(related='category_id.is_pos_groupable')
+
+
+class ProductProduct(models.Model):
+    _inherit = 'product.product'
+
+    def get_pos_anglo_saxon_price_unit(self, partner_id, quantity, order_ids):
```

Comment:
- Define a private method
- There is no need to define it on product, you can define it on the `pos.order`, or maybe `pos.order.line`